config.toml.sample

```diff
@@ -9,6 +9,12 @@
     admin_username = "listmonk"
     admin_password = "listmonk"
 
+[tls]
+    enable = true
+    domain = "listmonk.company.com"
```

Comment:
This would be duplicate info that is already entered in the UI and saved in the database, but I don't think this setting should go here at all or where it is either. See #346 for why.